src/main/java/uk/gov/digital/ho/hocs/templates/application/RestHelper.java

```diff
@@ -35,14 +33,12 @@ public RestHelper(RestTemplate restTemplate, @Value("${hocs.basicauth}") String
         this.requestData = requestData;
     }
 
-    @Retryable(maxAttemptsExpression = "${retry.maxAttempts}", backoff = @Backoff(delayExpression = "${retry.delay}"))
     public <R> R get(String serviceBaseURL, String url, Class<R> responseType) {
         log.info("RestHelper making GET request to {}{}", serviceBaseURL, url, value(LogEvent.EVENT, REST_HELPER_GET));
         ResponseEntity<R> response = restTemplate.exchange(String.format("%s%s", serviceBaseURL, url), HttpMethod.GET, new HttpEntity<>(null, createAuthHeaders()), responseType);
         return response.getBody();
     }
 
-    @Retryable(maxAttemptsExpression = "${retry.maxAttempts}", backoff = @Backoff(delayExpression = "${retry.delay}"))
```

Comment:
Can you remove `retry.maxAttempts` and `retry.delay` from the app props?

Reply:
Is there a library that can be pulled now?

Reply:
no, they aren't in there?! maybe passed in from the deployment?!!!